src/hooks/requests/useProjects.ts

```diff
@@ -130,6 +130,9 @@ export const useUpdateProject = ({ id }: { id?: string }) => {
           return { data: newData }
         }
       )
+      map(data.sub_projects, (subProject) =>
```

Comment:
**[blocker]**

We should avoid refetching whenever possible. This should only be for cases, where we defiitely need it and have no other way. This one can cause a lot of refetching (1 API request per sub_project).

Inside the response we already have the updated subProjects, so we could just use that to update our cache, instead of refetching.

Extra note* current solution doesn't seem to work, since the new deadline_at value from the subproject is not added to the form. It will still show the old deadline for the subproject.